cmd/lnmuxd/config.go

```diff
@@ -39,11 +39,22 @@ type Config struct {
 	// and eventually pprof and/or other instrumentation listen on.
 	InstrumentationAddress string `yaml:"instrumentationAddress"`
 
-	Logging LoggingConfig `mapstructure:"logging" json:"logging" yaml:"logging"`
+	Logging LoggingConfig `yaml:"logging"`
 }
 
 // LoggingConfig contains options related to log outputs.
 type LoggingConfig struct {
+	// Level defined the minimum log level to be output
+	Level string `yaml:"level"`
+
+	// Format defined the output format:
+	//  'console' will be human-readable
+	//  'json' will be machine-readable.
+	Format string `yaml:"format"`
+
+	// WithCaller indicates if the caller is displayed in the log
+	WithCaller bool `yaml:"withCaller"`
```

Comment:
I used this field to keep the actual behaviour as it is.

But if it is not a problem to display the caller as well in the console (when debugging) then I can remove it

Reply:
I like having the option.